MetaMask/metamask-mobile
# feat(ramp): offramp unsupported region copy update

## **Description**

Very small copy change to the ramp regions page to support new Sell copy requirements

## **Related issues**
N/A

## **Manual testing steps**
Check the sell version of the region page and verify that the copy has been updated

## **Screenshots/Recordings**

### **Before**

### **After**

![Screenshot 2023-11-08 at 11 41 48 AM](https://github.com/MetaMask/metamask-mobile/assets/15269424/45a82130-7f0b-4ad1-9717-426a862694b7)


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
